sormas-ui/src/main/java/de/symeda/sormas/ui/symptoms/SymptomsForm.java

```diff
@@ -788,6 +789,35 @@ protected void addFields() {
 			}
 		}, ValoTheme.BUTTON_LINK);
 
+
+		//setting symptoms to unknown button
```

Comment:
Please extract the code to a dedicated method an reuse it for the two functionalities (setting fields to no and settings fields to unknown) to reduce the amount of duplicate code.